ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -957,6 +1043,163 @@ def wait_until_cpu_port_up(self):
                 break
             time.sleep(self.TIMEOUT)
 
+    def send_in_background(self, packets_list = None, interval = None):
+        """
+        This method sends predefined list of packets with predefined interval.
+        """
+        if not interval:
+            interval = self.send_interval
+        if not packets_list:
+            packets_list = self.packets_list
+        sender_start = datetime.datetime.now()
+        self.log("Sender started at %s" % str(sender_start))
+        for entry in packets_list:
+            time.sleep(interval)
```

Comment:
I'd remove this. python is slow enough

Reply:
Python is slow, but is able to send ~500 packet/second.
Its good for precision, but not for scapy.

The advisable time to detect warm-reboot disrupt is ~180 sec.
In this case:
180*500*2 = 180 000 packets (in and out).

Besides, after the warm-reboot the swtich becomes flat and floods packets for ~10 seconds out of 32 ports:
+ 10*500*30 = 150 000 packets more.

I added inter-packet interval to slow down the speed, sacrificing precision to ~5-10ms).
It made capture twice less.

To overcome this case I would consider another approach:
Make the Sniffer sniff only on selected 5 ports (1 vlan + 4 Lag), it will get rid of floods.
But in this case Sniff must be split to 5 threads (as scapy.sniff accepts one port). Then combine all captures and analize them.

But I think its a matter of newer design.